### PR TITLE
deps: update js-cookie to v3

### PR DESCRIPTION
**Pull request by chambo-e**
Hello 👋 

A small PR to bump js-cookie to v3

### CODE REVIEW

File: src/core/user/index.ts [hunk 1]
```diff
@@ -87,32 +87,46 @@ export class Cookie extends Store {
...
+
+      if (!value) {
+        return null
+      }
+
+      try {
+        return JSON.parse(value)
+      } catch (e) {
+        return (value as unknown) as T
+      }
```
Comment (arielsilvestri) [inline src/core/user/index.ts:111]: Just curious - how do we handle JSON we can't parse later on, or is it not a big deal?

**Review by arielsilvestri — APPROVED** [review]
lgtm

File: src/core/user/index.ts (hunk 1 above)
Comment (chambo-e) [reply src/core/user/index.ts:111]: I assumed that JSON that can't be parsed is "not" JSON 😄 

I kinda stole the logic from the previous implementation in js-cookie ([here](https://github.com/js-cookie/js-cookie/blob/54962f884e9ae33f93e13ac903ffaf1d5a523598/src/js.cookie.js#L125))

**Comment by pooyaj** [comment]
merging to a local branch to allow the CI run the test suites

**Pull request merged**